src/Features/LanguageServer/Protocol/Handler/Diagnostics/DocumentPullDiagnosticHandler.cs

```diff
@@ -75,17 +75,24 @@ protected override DiagnosticTag[] ConvertTags(DiagnosticData diagnosticData, bo
             if (category == PullDiagnosticCategories.Task)
                 return new(GetDiagnosticSources(diagnosticKind: default, nonLocalDocumentDiagnostics: false, taskList: true, context, GlobalOptions));
 
-            var diagnosticKind = category switch
+            DiagnosticKind? diagnosticKind;
+            if (category == PullDiagnosticCategories.DocumentCompilerSyntax)
```

Comment:
>if (category == PullDiagnosticCategories.DocumentCompilerSyntax) [](http://example.com/codeflow?start=12&length=64)

Necessary because this isn't a const